src/Features/Core/Portable/SolutionCrawler/InternalSolutionCrawlerOptions.cs

```diff
@@ -18,7 +18,7 @@ internal static class InternalSolutionCrawlerOptions
         public static readonly Option2<bool> DirectDependencyPropagationOnly = new Option2<bool>(nameof(InternalSolutionCrawlerOptions), "Project propagation only on direct dependency", defaultValue: true,
             storageLocations: new LocalUserProfileStorageLocation(LocalRegistryPath + "Project propagation only on direct dependency"));
 
-        public static readonly Option2<int> ActiveFileWorkerBackOffTimeSpanInMS = new Option2<int>(nameof(InternalSolutionCrawlerOptions), "Active file worker backoff timespan in ms", defaultValue: 400,
+        public static readonly Option2<int> ActiveFileWorkerBackOffTimeSpanInMS = new Option2<int>(nameof(InternalSolutionCrawlerOptions), "Active file worker backoff timespan in ms", defaultValue: 100,
```

Comment:
@tmat @CyrusNajmabadi This is one place where we can experiment. This is the active file analysis delay, which can now be reduced as actual execution doesn't happen in OOP. Talking to @heejaechang offline, we also have a possibility of completely removing the delay. I have taken a middle path and reduced the delay, but still not completely removed it. I will perform some RPS runs to compare perf numbers with different values here.

Reply:
It seems like this won't be necessary once we move to LSP diagnostics and pull model. The client will decide how often to ask for diagnostics.

---
In reply to: [449795559](https://github.com/dotnet/roslyn/pull/45607#discussion_r449795559) [](ancestors = 449795559)

Reply:
Is the idea that we will completely remove the solution crawler? Otherwise, we need some kind of delay for running the background analyses based on solution crawler.

Reply:
Yes, we might remove it completely.